client/components/timezone/index.jsx

```diff
@@ -15,9 +16,32 @@ import QueryTimezones from 'components/data/query-timezones';
 import getRawOffsets from 'state/selectors/get-raw-offsets';
 import getTimezones from 'state/selectors/get-timezones';
 
+function maybeFindLinkedTimezone( timezone, timezones ) {
+	if ( ! timezones || timezone in timezones ) {
```

Comment:
I'm not sure if this works as intended. For example, I get the default suggestion on page load as Guernsey instead of London:

<img width="591" alt="Screenshot 2020-01-10 at 5 12 15 PM" src="https://user-images.githubusercontent.com/1269602/72150812-7c4d3100-33cc-11ea-80e0-47bdf1a6547f.png">